src/components/organizations/shared/MemberAddForm/httpRequestSelector.js

```diff
@@ -0,0 +1,33 @@
+import { createSelector } from 'reselect';
+
+const specificHttpRequestSelector = state => state.httpRequests
+const organizationIdSelector = (state, props) => props.organizationId
+
+function isExistingMember(request) {
+  return !!_.get(request, 'response.hasUser')
+}
+
+export const httpRequestSelector = createSelector(
+  specificHttpRequestSelector,
+  organizationIdSelector,
+  (httpRequests, organizationId) => {
+
+    let relevantRequests = httpRequests.filter(r => (
+      (r.metadata.organizationId === organizationId) &&
+      (r.entity === 'userOrganizationMembershipCreate')
+    ))
+
+    let formattedRequests = relevantRequests.map(r => ({
```

Comment:
consts. Also rename your input var to allRequests and this var to httpRequests and you can return it with {httpRequests}. Could also return inline.